README.md

```diff
@@ -102,4 +102,36 @@ When you have implemented the two (or more) classes, describe the complexity of
 
 **FIXME: OPERATION COMPLEXITY**
 
+| Function | ListGenome | linkedlistGenome |
+| --- | --- | --- |
+| init | O(n) | O(n) |
+| insert_te | O(m) + O(disable_te)  | O(k+m) + O(disable_te |
```

Comment:
For the `list` implementation, you do a lot of copying, so it is actually $O(n+m)$ plus the call to `disable_te`. Otherwise, it is correct.